<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>profile</class>
 <widget class="QWidget" name="profile">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1050</width>
    <height>652</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <pointsize>10</pointsize>
   </font>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QGroupBox" name="groupBox">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>130</y>
     <width>341</width>
     <height>491</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(255, 252, 213);</string>
   </property>
   <property name="title">
    <string>GroupBox</string>
   </property>
   <widget class="QLabel" name="lbl_user_pro">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>30</y>
      <width>231</width>
      <height>18</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="text">
     <string>username</string>
    </property>
   </widget>
   <widget class="QLabel" name="lbl_name_pro">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>80</y>
      <width>221</width>
      <height>16</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="text">
     <string>name</string>
    </property>
   </widget>
   <widget class="QLabel" name="lbl_age_pro">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>130</y>
      <width>191</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="text">
     <string>birthday</string>
    </property>
   </widget>
   <widget class="QLabel" name="lbl_country_pro">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>180</y>
      <width>221</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="text">
     <string>country</string>
    </property>
   </widget>
   <widget class="QLabel" name="lbl_link_pro">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>280</y>
      <width>241</width>
      <height>16</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="text">
     <string>link</string>
    </property>
   </widget>
   <widget class="QLabel" name="lbl_phone_pro">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>230</y>
      <width>261</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="text">
     <string>phone number</string>
    </property>
   </widget>
   <widget class="QTextBrowser" name="textBrowser_bio_pro">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>330</y>
      <width>131</width>
      <height>111</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(255, 235, 156);</string>
    </property>
    <property name="html">
     <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'MS Shell Dlg 2'; font-size:7.8pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;span style=&quot; font-size:10pt;&quot;&gt;bio&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
   </widget>
  </widget>
  <widget class="QPushButton" name="pushButton">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>50</y>
     <width>161</width>
     <height>61</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton {
color: #333;
border: 2px solid #333;
border-radius: 20px;
border-style: outset;
background: qradialgradient(
cx: 0.3, cy: -0.4, fx: 0.3, fy: -0.4,
radius: 1.35, stop: 0 #fff, stop: 1 #ffff7f
);
padding: 5px;
}

QPushButton:hover {
background: qradialgradient(
cx: 0.3, cy: -0.4, fx: 0.3, fy: -0.4,
radius: 1.35, stop: 0 #fff, stop: 1 #bbb
);
}</string>
   </property>
   <property name="text">
    <string>Setting</string>
   </property>
  </widget>
  <widget class="QGroupBox" name="groupBox_searchhash_pro">
   <property name="geometry">
    <rect>
     <x>410</x>
     <y>460</y>
     <width>261</width>
     <height>151</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(255, 210, 253);</string>
   </property>
   <property name="title">
    <string>search #</string>
   </property>
   <widget class="QLabel" name="lbl_searchid_pro_2">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>30</y>
      <width>131</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Enter hashtag</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="ln_searchhash_pro">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>70</y>
      <width>221</width>
      <height>22</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(255, 255, 255);</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_searchhash_pro">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>100</y>
      <width>93</width>
      <height>28</height>
     </rect>
    </property>
    <property name="text">
     <string>search</string>
    </property>
   </widget>
  </widget>
  <widget class="QGroupBox" name="groupBox_follow_pro">
   <property name="geometry">
    <rect>
     <x>410</x>
     <y>290</y>
     <width>261</width>
     <height>151</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(206, 253, 255);</string>
   </property>
   <property name="title">
    <string>follow</string>
   </property>
   <widget class="QLabel" name="lbl_follow_pro">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>30</y>
      <width>131</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Enter Username</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="ln_follow_pro">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>70</y>
      <width>221</width>
      <height>22</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(255, 255, 255);</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_follow_pro">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>100</y>
      <width>93</width>
      <height>28</height>
     </rect>
    </property>
    <property name="text">
     <string>follow</string>
    </property>
   </widget>
  </widget>
  <widget class="QGroupBox" name="groupBox_like_pro">
   <property name="geometry">
    <rect>
     <x>410</x>
     <y>130</y>
     <width>261</width>
     <height>151</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(161, 222, 255);</string>
   </property>
   <property name="title">
    <string>like</string>
   </property>
   <widget class="QLabel" name="lbl_like_pro">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>30</y>
      <width>131</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Enter Username</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="ln_like_pro">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>70</y>
      <width>221</width>
      <height>22</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(255, 255, 255);</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_like_pro">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>100</y>
      <width>93</width>
      <height>28</height>
     </rect>
    </property>
    <property name="text">
     <string>follow</string>
    </property>
   </widget>
  </widget>
  <widget class="QGroupBox" name="groupBox_searchid_pro">
   <property name="geometry">
    <rect>
     <x>690</x>
     <y>470</y>
     <width>251</width>
     <height>141</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(255, 204, 179);</string>
   </property>
   <property name="title">
    <string>other profile</string>
   </property>
   <widget class="QLabel" name="lbl_searchid_pro">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>30</y>
      <width>131</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Enter Username</string>
    </property>
    <property name="buddy">
     <cstring>ln_searchid_pro</cstring>
    </property>
   </widget>
   <widget class="QLineEdit" name="ln_searchid_pro">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>70</y>
      <width>221</width>
      <height>22</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(255, 255, 255);</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_searchid_pro">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>100</y>
      <width>93</width>
      <height>28</height>
     </rect>
    </property>
    <property name="text">
     <string>search</string>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>btn_searchid_pro</sender>
   <signal>clicked()</signal>
   <receiver>ln_searchid_pro</receiver>
   <slot>clear()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>789</x>
     <y>584</y>
    </hint>
    <hint type="destinationlabel">
     <x>790</x>
     <y>560</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
